app/views/shared/forms/_types_of_income_form.html.erb

```diff
@@ -33,20 +37,22 @@
     </fieldset>
   <% end %>
 
-  <details class="govuk-details govuk-!-margin-top-4" data-module="govuk-details">
-    <summary class="govuk-details__summary">
-          <span class="govuk-details__summary-text">
-            <%= t('.expanded_explanation.heading') %>
-          </span>
-    </summary>
+  <% if citizen %>
```

Comment:
Could you just do `<% if journey == :citizen %>` here, and then the remove the code on lines 1-4? You're not using the `citizen` variable anywhere else so it seems a bit unncessary.